odoo/addons/test_new_api/tests/test_one2many.py

```diff
@@ -88,11 +90,24 @@ def test_rpcstyle_one_by_one(self):
         for name in range(10):
             self.multi.lines = [(0, 0, {"name": str(name)})]
         self.operations()
+        self.multi.unlink()
+        self.multi = self.env["test_new_api.multi"].new({
+            "name": "What is up?"
+        })
+        for name in range(10):
+            self.multi.lines = [(0, 0, {"name": str(name)})]
+        self.operations(True)
```

Comment:
Please make a new test method `test_rpcstyle_one_by_one_on_new` with the 6 lines above.